tests/arrays/alignment_aarch64_linux.c3

```diff
@@ -0,0 +1,14 @@
+function main : () -> int = {
+    // Arrays use the alignment of their elements.
+    let small : int[3];
+
+    // Irrespective of the total array size.
+    let large : int[4];
+
+    return 0;
+}
+
+/// @FOR aarch64_linux
```

Comment:
Ooo, smart. Depending on how many of these we want, a target directory might be nice. This is good for now tho.

Reply:
Yeah, I know. I just didn't feel like replicating all the subdirectories etc multiple times. If we have too many of them we'll make separate directories.